capnpy/testing/compiler/test_import.py

```diff
@@ -50,6 +50,7 @@ def test_import(self):
             b @1 :P.Point;
             c @2 :List(P.Point);
             d @3 :List(P.Color);
+            e @4 :P.Color;
```

Comment:
I think this test is getting too big and testing too many stuff at once. Maybe it's better to split into many?